### DISCUSSION TITLE
Mirth db is taking a lot of space

### DISCUSSION THREAD

**Discussion by tGitm** [nextgenhealthcare/connect#5707] [Q&A]
Hi guys 👋 

Our customers are reporting to us, that their mirth folder is taking up a lot of space (80, 110, 100GB). When I dig into the Mirt Connect folder I found that the problem is mirthdb/seq0 folder (check image below - image from customer).

![image](https://user-images.githubusercontent.com/47832737/222671430-54b04169-502b-4542-8556-0fd1c501f17b.png)

How can we fix this problem with space?
Thanks for helping me out!

**Comment by ObsessiO** [comment]
First of all, Mirth Connect's default database, Apache Derby, is included only to help you set up quickly. Because it is not a production-level database, Nextgen Healthcare strongly recommends that you not use Derby for production.

Secondly, please check message pruning settings at channels. These settings determine how long to keep message data around before automatically removing it with Data Pruner. Please also check Data Pruner Settings tab at the Settings for the enable, schedule types and prune settings.

**Reply by tGitm** [reply]
Thanks! I've had development channel instead of production settings, but how can I clean the db now?

**Reply by pacmano1** [reply]
Just to be clear, you should not be using Derby in production at all.  You can revise pruning settings to remove old messages and metadata via the number of day settings on the right hand side of the screen.  Also in system settings you need to make sure pruning is enabled.   See https://db.apache.org/derby/docs/10.7/adminguide/cadminspace21579.html for understanding how to clean Derby space.  If you do not care about message history you can also delete and reimport channels on real database engines but I don't know if that is true for Derby.  That should be trivial for you to test.
